11/zkan/generators.py

```diff
@@ -0,0 +1,50 @@
+"""
+Turn the following unix pipeline into Python code using generators
+
+$ for i in ../*/*py; do grep ^import $i|sed 's/import //g' ; done | sort | uniq -c | sort -nr
+   4 unittest
+   4 sys
+   3 re
+   3 csv
+   2 tweepy
+   2 random
+   2 os
+   2 json
+   2 itertools
+   1 time
+   1 datetime
+"""
+from collections import Counter
+import glob
+import re
+
+
+def gen_files(pat):
+    yield from glob.glob(pat)
+
+
+def gen_lines(files):
+    for each in files:
+        with open(each) as f:
```

Comment:
have to double-check if we used it but starting (from memory) 3.3 you can use `yield from`